### ISSUE TITLE
[css-scrollbars] Add `thick` value to `scrollbar-width`

### ISSUE THREAD

**Issue by SebastianZ** [w3c/csswg-drafts#10650]
As [noted in the specification](https://drafts.csswg.org/css-scrollbars-1/#scrollbar-width), authors should generally refrain from overwriting the scrollbar width besides supporting specific UX goals. And @tabatkins [pointed out](https://github.com/w3c/csswg-drafts/issues/6263#issuecomment-855015739) that implementers object to add support for custom widths.

Though as outlined by several people in #6263, there's the use case of accessibility for having wider scrollbars.

The `auto` refers to the OS or UA settings, which is ok for most cases. Though if neither the OS nor the UA provide settings to change the width, users are stuck with the default width as the maximum.
Also, authors may want to emphasize the possibility to scroll and element.

Therefore, I suggest to add a new value `thick` to `scrollbar-width` that aligns with the definition of `thin` but makes the width thicker than `auto`.

The concrete spec. text might read like this:

`thick`
Implementations should use thicker scrollbars than `auto`. This may mean a thick variant of scrollbar provided by the platform, or a custom scrollbar thicker than the default platform scrollbar.

Sebastian

**Comment by nt1m** [comment]
I would object to this. Not all system frameworks support rendering scrollbars at thick sizes.

I would argue it's up to `auto` and the UA to consistently provide a decent UI for accessibility purposes, not the web author to randomly increase the size of the scrollbar (I doubt many people want this from an UX perspective).

**Comment by SebastianZ** [comment]
Yes, it's generally up to the UA/OS to provide a decent UI. The issue is that they sometimes don't. 

And this is primarily a way for users to adjust this via a user style sheet even when the OS or UA doesn't provide a UI setting for this. It is less useful for web authors.

> ... not the web author to randomly increase the size of the scrollbar (I doubt many people want this from an UX perspective).

Bad UX may also be a counter-argument for the `thin` value, though it was specced and is already widely implemented.

> I would object to this. Not all system frameworks support rendering scrollbars at thick sizes.

If they don't, UAs should render custom scrollbars if possible. If not, they might fall back to `auto`.

Sebastian
